docs/release_notes/v1.8.5.md

```diff
@@ -0,0 +1,15 @@
+# Dapr 1.8.5
+
+## Fixes an issue in Kafka Pubsub preventing the component from initializing with SALS password auth
+
+### Problem
+
+The Kafka Pubsub component failed to initialize when using SASL password authentication.
+
+### Root cause
```

Comment:
Missing "Impact". See example: https://github.com/dapr/dapr/blob/master/docs/release_notes/v1.8.3.md

Reply:
That's an optional section. It's not missing. We didn't always have that section.